Libraries/LibWeb/Layout/InlineLevelIterator.cpp

```diff
@@ -491,20 +492,33 @@ Optional<InlineLevelIterator::Item> InlineLevelIterator::next_without_lookahead(
         return {};
 
     if (is<Layout::TextNode>(*m_current_node)) {
-        auto& text_node = static_cast<Layout::TextNode const&>(*m_current_node);
+        auto const& text_node = static_cast<Layout::TextNode const&>(*m_current_node);
```

Comment:
The changes to InlineLevelIterator are separate enough that they belong in their own commit. (Generally, if you end up putting "and" or "also" in a commit message, it's a clue that it might want to be multiple commits. Sometimes that's not possible though.)